charts/prometheus-node-exporter/templates/_helpers.tpl

```diff
@@ -58,8 +58,13 @@ Create the name of the service account to use
 The image to use
 */}}
 {{- define "prometheus-node-exporter.image" -}}
+{{- if .Values.image.sha -}}
+"{{ .Values.image.repository }}:{{ .Values.image.tag }}@sha256:{{ .Values.image.sha }}"
```

Comment:
I believe this line got added accidentally?

Reply:
No, this is intentional, I want to check for a sha and add the image in that fashion if it is found.

Reply:
Ohh I see, sorry, I was grabbing it as an example of the format to use and forgot to delete the line.